shell/browser/osr/osr_web_contents_view.h

```diff
@@ -75,6 +75,7 @@ class OffScreenWebContentsView : public content::WebContentsView,
   void UpdateWindowControlsOverlay(const gfx::Rect& bounding_rect) override {}
   content::BackForwardTransitionAnimationManager*
   GetBackForwardTransitionAnimationManager() override;
+  void DestroyBackForwardTransitionAnimationManager() override;
```

Comment:
```suggestion
  void DestroyBackForwardTransitionAnimationManager() override {}
```

Reply:
My suggested change would be more consistent with the current file, but either approach works.

Reply:
implementing these empty `{}` functions in the header is generally how Chromium does it too